### ISSUE TITLE
Upgrading flyway from 2.1.1 to 4.0.3 - repeatable migrations DB2 index error

### ISSUE THREAD

**Issue by garrmark** [flyway/flyway#1586]
###### What version of Flyway are you using?

4.0.3

###### Which client are you using? (Command-line, Java API, Maven plugin, Gradle plugin, SBT plugin, ANT tasks)

Command-line

###### What database are you using (type & version)?

DB2 LUW 10.5

###### What operating system are you using?

Windows 10

###### What did you do?
(Please include the content causing the issue, any relevant configuration settings, and the command you ran)

I'm trying out the new repeatable migrations, but there appears to still be a unique index on the flyway version table.  This causes flyway migrate to fail with `ERROR: Unable to insert row for version 'null' in metadata table`

The offending index appears to be:
`CREATE UNIQUE INDEX DATA_ACCESS."FLYWAY_VERSION_pk"
  ON DATA_ACCESS.FLYWAY_VERSION
    ( "version" ASC )
  ALLOW REVERSE SCANS
  COMPRESS NO
  INCLUDE NULL KEYS;`

###### What did you expect to see?

flyway migrate working successfully

###### What did you see instead?

ERROR: Unable to insert row for version 'null' in metadata table

**Comment by axelfontaine** [comment]
No idea how you got this. We actually properly drop the primary key constraint during the upgrade. See https://github.com/flyway/flyway/blob/master/flyway-core/src/main/resources/org/flywaydb/core/internal/dbsupport/db2/upgradeMetaDataTable.sql#L20

Did we miss something?

**Comment by garrmark** [comment]
I'm not sure, our metadata table has been around since version 2, I thought that maybe at some point the Index was also created by the code.  Our current production DB which is still using flyway 2.1.1 has three indexes TABLE_pk, TABLE_vr_idx, TABLE_ir_idx, TABLE_s_idx.  The TABLE_pk is a unique index and has to be dropped even though the constraint on the table was changed.  That's what I did to fix it, but I thought that it might be good to check for it during the migrate.
